examples/common/passive/FairGeoCave.cxx

```diff
@@ -17,109 +17,121 @@
 
 #include "FairGeoCave.h"
 
-#include "FairGeoBasicShape.h"          // for FairGeoBasicShape
-#include "FairGeoMedia.h"               // for FairGeoMedia
-#include "FairGeoMedium.h"              // for FairGeoMedium
-#include "FairGeoNode.h"                // for FairGeoNode, etc
-#include "FairGeoShapes.h"              // for FairGeoShapes
+#include "FairGeoBasicShape.h"   // for FairGeoBasicShape
+#include "FairGeoMedia.h"        // for FairGeoMedia
+#include "FairGeoMedium.h"       // for FairGeoMedium
+#include "FairGeoNode.h"         // for FairGeoNode, etc
+#include "FairGeoShapes.h"       // for FairGeoShapes
 
-#include <TList.h>                      // for TList
-
-#include <string.h>                     // for strcmp
-#include <iostream>                     // for cout
-#include <fstream>                      // for fstream
+#include <TList.h>    // for TList
+#include <fstream>    // for fstream
+#include <iostream>   // for cout
+#include <string.h>   // for strcmp
 
 using namespace std;
-ClassImp(FairGeoCave)
+ClassImp(FairGeoCave);
 
 FairGeoCave::FairGeoCave()
-  : FairGeoSet(),
-    name("cave")
+    : FairGeoSet()
+    , name("cave")
 {
-  fName = "cave";
-  name = "cave";
-  maxModules=1;
+    fName = "cave";
+    name = "cave";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'name' is assigned in constructor body. Consider performing initialization in initialization list.](https://app.codacy.com/manual/FairRootGroup/FairRoot/pullRequest?prid=5323596)